.stylelintrc

```diff
@@ -1,18 +1,16 @@
 {
-	"extends": "stylelint-config-standard",
-	"plugins": ["stylelint-scss", "stylelint-order"],
+	"extends": [
+		"stylelint-config-standard-scss",
+		"stylelint-prettier/recommended"
+	],
+	"plugins": ["stylelint-order"],
 	"rules": {
+		"selector-class-pattern": "^[a-z][a-zA-Z0-9]+$",
+		"scss/percent-placeholder-pattern": "^[a-z][a-zA-Z0-9]+$",
+		"scss/at-extend-no-missing-placeholder": null,
```

Comment:
This needs reinstating but I've added it initially to get it to pass whilst I fix other things